src/main/java/com/kobylynskyi/graphql/codegen/model/MappingConfig.java

```diff
@@ -39,6 +39,7 @@
     private String subscriptionReturnType;
     private Boolean generateEqualsAndHashCode;
     private Boolean generateToString;
+    private Boolean asyncApi;
```

Comment:
I'd rather use a verb here, like `useAsyncApi` or `generateAsyncApi`.

Reply:
Vote for `generateAsyncApi`

Reply:
Done!